interfaces/main.py

```
@@ -177,6 +178,9 @@ async def startup_event():
     with startup_sqlite_writes_bypass_queue():
         _stop_all_running_novels()
 
+    # 清理上次运行时残留的 WAL（PASSIVE 模式不阻塞，安全执行）
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)